workflows/deepfactor-scan-image.yml

```
@@ -0,0 +1,28 @@
+name: Deepfactor scan image
+
+on:
+  workflow_call:
+    inputs:
+      image:
+        description: The name of the image to deploy, should already be built
+        required: true
+        type: string
+    secrets:
+      deepfactor_run_token:
+        description: The deepfactor run token
+        required: true
+
+
+jobs:
+  scan:
+    name: Deepfactor scan image
+    runs-on: ubuntu-latest
+    container: public.ecr.aws/deepfactor/df/runtime:latest
+    env:
+      DF_RUN_TOKEN: ${{ secrets.DF_RUN_TOKEN }}
```

Should this match line 11 above?
```suggestion
      DF_RUN_TOKEN: ${{ secrets.deepfactor_run_token }}
```